src/u/js/main.js

```diff
@@ -1,5 +1,31 @@
-$(document).ready(function() {
+$(document).ready(function () {
     $('#example').DataTable();
     $('div.alert').delay(3000).fadeOut(500);
-} );
-$('[data-toggle="tooltip"]').tooltip();
\ No newline at end of file
+
+
+
+});
+$('[data-toggle="tooltip"]').tooltip();
+
+
+function makeid() {
```

Comment:
Encase we need this useful function again, have a parameter for the id length

Reply:
this has been added